src/test/java/com/shapesecurity/shift/es2016/test262/Test262.java

```diff
@@ -0,0 +1,338 @@
+
+			// trailing commas not in ES6
+			"src/test/resources/test262/test/language/statements/class/definition/params-trailing-comma.js",
+			"src/test/resources/test262/test/language/statements/class/definition/params-trailing-comma-length.js",
+			"src/test/resources/test262/test/language/statements/function/params-trailing-comma.js",
+			"src/test/resources/test262/test/language/statements/function/params-trailing-comma-length.js",
+			"src/test/resources/test262/test/language/expressions/object/method-definition/params-trailing-comma-length.js",
+			"src/test/resources/test262/test/language/expressions/object/method-definition/params-trailing-comma.js",
+			"src/test/resources/test262/test/language/expressions/arrow-function/params-trailing-comma-length.js",
+			"src/test/resources/test262/test/language/expressions/arrow-function/params-trailing-comma.js",
+			"src/test/resources/test262/test/language/expressions/function/params-trailing-comma-length.js",
+			"src/test/resources/test262/test/language/expressions/function/params-trailing-comma.js",
+
+			// broken test
+			"src/test/resources/test262/test/language/module-code/namespace/internals/set-prototype-of-null.js",
+
+			// shift-java#207
+			"src/test/resources/test262/test/language/expressions/tagged-template/invalid-escape-sequences.js"
+
+	));
+
+	private static final HashSet<String> xfailEarlyErrors = new HashSet<>(Arrays.asList(
+
+			// exponentiation early errors not implemented
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-delete-unary-expression-base.js",
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-typeof-unary-expression-base.js",
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-logical-not-unary-expression-base.js",
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-void-unary-expression-base.js",
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-negate-unary-expression-base.js",
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-plus-unary-expression-base.js",
+			"src/test/resources/test262/test/language/expressions/exponentiation/exp-operator-syntax-error-bitnot-unary-expression-base.js",
+
+			// yield not implemented
+			"src/test/resources/test262/test/language/expressions/assignment/dstr-obj-id-identifier-yield-expr.js",
+			"src/test/resources/test262/test/language/statements/for-in/dstr-obj-id-identifier-yield-expr.js",
+			"src/test/resources/test262/test/language/statements/for-of/dstr-obj-id-identifier-yield-expr.js",
+
+			// shift-java#208
+			"src/test/resources/test262/test/language/global-code/decl-lex-restricted-global.js"
+	));
+
+	private static final HashSet<String> skip = new HashSet<>(Arrays.asList(
```

Comment:
I am morally opposed to skipped tests. They should be xfailed, not skipped. Otherwise the expectations file gets stale as new features are implemented.

Reply:
Removed all skipped tests in favor of xfailing